.github/workflows/release.yml

```diff
@@ -39,10 +42,20 @@ jobs:
           name: ${{ inputs.files }}
         if: ${{ inputs.files != '' }}
 
+      # the token is needed so the github app can push to the repository
+      # the github app can bypass the branch protection rule (need a PR to merge)
+      # this way it can directly commit to main
+      - name: Obtain a GitHub App Installation Access Token
+        id: githubAppAuth
+        run: |
+          TOKEN="$(npx obtain-github-app-installation-access-token ci ${{ secrets.GH_BRANCH_PROTECTION_APP_TOKEN }})"
+          echo token=${TOKEN} >> $GITHUB_OUTPUT
```

Comment:
can you add a mask forthe token, so it does not leak by accident? (https://github.com/dtinth/obtain-github-app-installation-access-token/blob/31fa7d82cdda448c331dd089b8b99ed30cba4ec9/.github/workflows/test.yml#L14-L16C43)